about.html

```diff
@@ -0,0 +1,124 @@
+      href="https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Nanum+Gothic&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/1f634f9b31.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="about.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <article class="desktop-only">
+          <ul>
+            <li>
+              <a href="https://www.facebook.com/"
+                ><img src="images/facebook.png" alt=""
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com/"
+                ><img src="images/twitter.png" alt=""
+              /></a>
+            </li>
+            <li><a href="#">English</a></li>
+          </ul>
+        </article>
+        <article class="navigation">
+          <div class="mobile-menu-btn">
+            <button type="button" class="btn">
+              <img src="images/mobile_menu.png" alt="" />
+            </button>
+          </div>
+          <div class="logo">
+            <a href="index.html"><img src="images/logo.png" alt="" /></a>
+          </div>
+          <section class="menus">
+            <ul class="nav-menu">
+              <li><a href="#highlights">Highlights</a></li>
+              <li id="live"><a href="#">presentation</a></li>
+            </ul>
+          </section>
+        </article>
+      </nav>
+    </header>
+    <main>
+      <section class="main">
+        <div class="greeting">"Hello, Sharing world!"</div>
+        <div class="title">
+          COMPUTER SCIENCE <br />
+          CONFERENCE 2022
+        </div>
+        <div class="message">
+          A fun festival where people from more than 80 countries who believe in
+          the value of sharing and openness and create positive change gather
+          and share ideas will be held in Cyprus in December.
+        </div>
+        <div class="contact-us">
+          <p>
+            Feel free to reach out to us for additional informations <br />
+            about the Computer Science Conference 2022
+          </p>
+          <a href="https://telegram.org/"
+            ><i class="fa-brands fa-telegram"></i> </a
+          ><a href="https://www.whatsapp.com/"
+            ><i class="fa-brands fa-whatsapp"></i></a
+          ><a href="https://www.viber.com/en/"
+            ><i class="fa-brands fa-viber"></i
+          ></a>
+          <p>+99999999999</p>
+        </div>
+      </section>
+    </main>
+    <section id="highlights">
+      <h2>Computer Science Conference 2021</h2>
+      <p>Highlights from the last Computer Science Conference.</p>
+      <img src="images/conf-1.jpg" alt="" /><img
+        src="images/conf-2.jpg"
+        alt=""
+      /><img src="images/conf-3.jpg" alt="" /><img
+        src="images/conf-4.jpg"
+        alt=""
```

Comment:
- [ ] Kindly make sure you stick to the provided design as much as possible. 
Consider styling these conference images as it is on the provided template.

Ypur design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197215392-03390171-2233-462f-ba93-78f696ff784e.png)

Provided template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197215555-9650f69d-0284-4d39-935a-671cd7100932.png)